registry-scanner/go.mod

```diff
@@ -3,20 +3,81 @@ module github.com/argoproj-labs/argocd-image-updater/registry-scanner
 go 1.22.3
```

Comment:
image-updater currently has go version 1.22.0. I think they should be consistent.